### ISSUE TITLE
Unstable Screenshare Recording

### ISSUE THREAD

**Issue by pybean** [ging/licode#366]
I just updated the licode server and it seems like something is wrong with recording screen. I changed two values in the licode_config.js file as below:
<pre>
config.erizoController.defaultVideoBW = 3000; //default value: 300
config.erizoController.maxVideoBW = 3000; //default value: 300
</pre>
Then I used additional 'videoSize' parameter on the client side for init stream:
<pre>
var stream = Erizo.Stream({screen:true, extensionId: 'my_extension_id', videoSize: [1280, 720, 1920, 1080]});
</pre>

Everything works fine with stream publishing and subscribing, but when I checked the recorded mkv files and the video stream is empty. The mkv file is completely filled by a blank audio stream. When I use ffmpeg to convert from mkv to webm with the '-an(no audio)' option, ffmpeg shows 'Output file is empty' warning.

In short:
1) video stream is dropped time to time
2) there shouldn't be an audio stream embedded in a screen recording file

Thanks.

**Comment by pybean** [comment]
I'm using the master branch by the way.

**Comment by lodoyun** [comment]
I just pushed a fix (b5f5322f5efca5a6b2f2d1b1e43448c4f7dd644a) that should help with this.
Please give it a try
